libbeat/metric/system/process/process.go

```diff
@@ -492,6 +562,17 @@ func (procStats *Stats) getSingleProcess(pid int, newProcs ProcsMap) *Process {
 		return nil
 	}
 
+	if procStats.EnableCgroups {
+		cgStats, err := procStats.cgroups.GetStatsForProcess(pid)
+		if err != nil {
+			procStats.logger.Debug("Error fetching cgroup data for process %s with pid=%d: %v", process.Name, process.Pid, err)
+			return nil
+		}
+		process.RawStats = cgStats
+		last := procStats.ProcsMap[process.Pid]
+		process.CgroupCPUTotalPct, process.CgroupCPUTotalPctNorm, process.CgroupCPUUserPct, process.CgroupCPUSystemPct, process.CgroupCPUUserPctNorm, process.CgroupCPUSystemPctNorm = GetCgroupPercentage(last, process)
```

Comment:
Returning so many values, specially of the same type, can be error-prone, and is difficult to read. Consider returning a struct instead, so you can do here:
```suggestion
		p := GetCgroupPercentage(last, process)
		process.CgroupCPUTotalPct = p.TotalPct
		process.CgroupCPUTotalPctNorm = p.TotalPctNorm
		process.CgroupCPUUserPct = p.UserPct
		process.CgroupCPUSystemPct = p.SystemPct
		process.CgroupCPUUserPctNorm = p.UserPctNorm
		process.CgroupCPUSystemPctNorm = p.SystemPctNorm
```